src/System.IO.IsolatedStorage/tests/System/IO/IsolatedStorage/ContainsUnknownFilesTests.cs

```diff
@@ -0,0 +1,85 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Reflection;
+using Xunit;
+
+namespace System.IO.IsolatedStorage
+{
+    public class ContainsUnknownFilesTests : IsoStorageTest
+    {
+        private static MethodInfo s_containsUnknownFilesMethod;
+
+        static ContainsUnknownFilesTests()
```

Comment:
Nit: the explicit cctor isn't needed here; you could just assign s_containsUnknownFilesMethod where it's declared